dotnet/csharp.bzl

```diff
@@ -529,4 +579,17 @@ def csharp_repositories(use_local_mono=False):
       # work when Workspaces import this using a repository rule.
       build_file = str(Label("//dotnet:nunit.BUILD")),
   )
+
+  native.new_http_archive(
+      name = "nuget",
+      url = "https://github.com/mono/nuget-binary/archive/0811ba888a80aaff66a93a4c98567ce904ab2663.zip", # Sept 6, 2016
```

Comment:
It would be awesome if we could use a bazel-mirror for this like we do for nunit.